### ISSUE TITLE
Captcha integration settings being cleared.

### ISSUE THREAD

**Issue by jwrudzin** [verbb/formie#262]
**Description**
We have noticed on several of our sites that the captcha integrations we set up have been cleared out and are no longer enabled.

**Steps to reproduce**
1. Enable any of the captcha integrations in Formie>Settings>Captchas
2. Navigate to another Formie settings page (for example the Formie>Submissions settings page) and click the save button.
3. Navigate back to the Formie>Settings>Captchas and the previously enabled captchas will be disabled (and any keys that were entered for Recaptcha will be cleared out as well).

**Form settings**
Form settings aren't relevant to this as it's a general settings issue and not related to a single form. One note though - when a captcha is enabled on a form, and then the captcha integrations are cleared out as outlined above, the captcha predictably disappears from the form integrations as well. However, when the captcha is re-enabled in Formie settings, the form-specific settings come back and are not cleared out.

**Additional context**
I just want to note that there may be additional things that are triggering these captcha integrations to be cleared out - the process I outlined above is just one way that I'm able to consistently reproduce this issue.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.4](https://github.com/verbb/formie/releases/tag/1.3.4)
